src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -299,6 +304,17 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
         // class itself to work with the generalized ConfigStore.
         const l1Token = args.key;
 
+        // For now, we'll need to check if this value is undefined. In the
+        // future, we should make this a required field.
+        if (parsedValue.uba !== undefined) {
+          // Parse and store UBA config
+          const ubaConfig = parseUBAConfigFromOnChain(parsedValue?.uba);
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
+          const { value: _value, key: _key, ...passedArgs } = args;
+          this.ubaConfigUpdates.push({ ...passedArgs, config: ubaConfig, l1Token });
+          sortEventsAscendingInPlace(this.ubaConfigUpdates);
```

Comment:
Why would you sort after each `.push`? Why not sort after going through all `ubaConfigUpdates`?

Reply:
Done: https://github.com/across-protocol/sdk-v2/pull/280/commits/c0664473b6fa3fb8f9a34cb0bb2d57aaab087244